File: ClientApp/src/app/interfaces/product/userInteract/response/purchaseDetailOptionResponse.ts
export interface PurchaseDetailOptionResponse {
    image: string;
    quantity: number;
    optionId: number;
    unitPrice: number;
    appliedId: number | null;
    totalPrice: number;
    createdAt: string;
    updatedAt: string | null;
    isReturned: boolean;
}

File: ClientApp/src/app/interfaces/Product/UserInteract/Response/PurchaseDetailOptionResponse.cs
namespace Contracts.Model.Product.UserInteract.Response;

public class PurchaseDetailOptionResponse : BaseResponse
{
	public string Image { get; set; }
	public int Quantity { get; set; }
	public Guid OptionId { get; set; }
	public double UnitPrice { get; set; }
	public Guid? AppliedId { get; set; }
	public bool IsReturned { get; set; }
}